### PR TITLE
Add Medium and LinkedIn to social partial

### PR DESCRIPTION
**Pull request by RicardoBelchior**
Hi there,

Simple PR to include LinkedIn and Medium accounts into the "social" partial social.html

Cheers,
Ricardo

### CODE REVIEW

**Comment by sauerj** [comment]
Nice contribution! 

I think it would be nice to split the changes into different commits. One for each button.

There is no configuration example for Medium in `config.toml.example`

@AngeloStavrow likes the buttons be alphabetical ordered, just like in #24

**Comment by RicardoBelchior** [comment]
sure , no problem

**Comment by AngeloStavrow** [comment]
Thanks for opening this PR! I'll have a look at it today.

File: layouts/partials/social.html [hunk 1]
```diff
@@ -35,6 +35,16 @@
       <a class="glyph" alt="Keybase profile" href="https://keybase.io/{{ .Site.Params.KeybaseUser }}"><img src="{{ .Site.BaseURL }}icons/keybase.svg" height="24px" width="24px"></a>
     </div>
     {{ end }}
+    {{ with .Site.Params.LinkedInUser }}
+    <div class="icon-24x24">
+      <a class="glyph" alt="LinkedIn profile" href="https://www.linkedin.com/in/{{ . }}"><img src="/icons/linkedin.svg" height="24px" width="24px"></a>
```
Comment (AngeloStavrow) [inline layouts/partials/social.html]: This is the way I had the social icons set up in the initial release of the theme, with `img src="/icons/icon-name.svg"`. @sauerj pointed out in #27 that if the Hugo site is set up somewhere other than a site's root directory (e.g., at `example.com/blog`), the icon assets won't be reachable, so I've changed it. For consistency, you can have a look at the way other icons are added in this file.

File: config.toml.example [hunk 2]
```diff
@@ -29,11 +30,11 @@ theme = "indigo"
 
   # These are parameters used for indieweb identity. You should set these along
   # with the above email/social network parameters.
-  [params.indieWeb]  
+  [params.indieWeb]
```
Comment (sauerj) [inline config.toml.example]: @AngeloStavrow in this commit there are some changes which have nothing to do with the PR. Are you okay by merging them? I noticed some trailing whitspaces in some files, but had not the time to fix this. Maybe a new issue is a good idea for this?

File: config.toml.example [hunk 3]
```diff
@@ -29,11 +30,11 @@ theme = "indigo"
...
     EmailAddress = "email.address@example.com"
     FlickrUser = "FlickrUserName"
     GitHubUser = "GitHubUserName"
     TwitterUser = "TwitterUserName"
     MicroBlogUser = "MicroBlogUserName"
     Country = "CountryName"
-    City = "CityName"
\ No newline at end of file
+    City = "CityName"
```
Comment (sauerj) [inline config.toml.example]: Same for this change as I mentioned above.

File: exampleSite/config.toml [hunk 4]
```diff
@@ -30,11 +31,11 @@ paginate = 3
 
   # These are parameters used for indieweb identity. You should set these along
   # with the above email/social network parameters.
-  [params.indieWeb]  
+  [params.indieWeb]
```
Comment (sauerj) [inline exampleSite/config.toml]: Same as above

File: exampleSite/config.toml [hunk 5]
```diff
@@ -30,11 +31,11 @@ paginate = 3
...
     EmailAddress = "email.address@example.com"
     FlickrUser = "FlickrUserName"
     GitHubUser = "GitHubUserName"
     TwitterUser = "TwitterUserName"
     MicroBlogUser = "MicroBlogUserName"
     Country = "CountryName"
-    City = "CityName"
\ No newline at end of file
+    City = "CityName"
```
Comment (sauerj) [inline exampleSite/config.toml]: Same as above.

**Comment by sauerj** [comment]
@RicardoBelchior sorry I forgot to submit my review this morning. I had the same suggestions as @AngeloStavrow

File: config.toml.example (hunk 2 above)
Comment (AngeloStavrow) [reply config.toml.example]: Opening a new issue is a good idea. I haven't really done a thorough audit of whitespace issues.

In the meanwhile I think it's fine to include these changes with this PR as it's really just a bit of cleanup, and not a change that affects functionality in some unrelated way.

**Comment by RicardoBelchior** [comment]
Sorry for the extra-spaces guys , I did wrote this PR without paying much attention^^
I updated the socials to the new format (my forked branch was not up to date).

And I force-pushed once again :) Let me know if you need any other change.

**Comment by RicardoBelchior** [comment]
Thanks guys 👍

**Comment by AngeloStavrow** [comment]
No, thank _you_, @RicardoBelchior, for your contributions! 🎉

**Pull request merged**